package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-gettext-parser",
+  "name": "@elser/react-gettext-parser",
```

Comment:
You can `reset --hard` to that commit and force push it up here too, if you want. Nice to keep a record, but no biggie!